include/pstl/internal/unseq_backend_simd.h

```diff
@@ -29,464 +29,470 @@
+template<class _Index, class _DifferenceType, class _Tp>
+_Index simd_fill_n(_Index __first, _DifferenceType __n, const _Tp& __value) noexcept {
 __PSTL_USE_NONTEMPORAL_STORES_IF_ALLOWED
 __PSTL_PRAGMA_SIMD
-    for (DifferenceType i = 0; i < n; ++i)
-        first[i] = value;
-    return first + n;
+    for (_DifferenceType __i = 0; __i < __n; ++__i)
+        __first[__i] = __value;
+    return __first + __n;
 }
 
-template<class Index, class DifferenceType, class Generator>
-Index simd_generate_n(Index first, DifferenceType size, Generator g) noexcept {
+template<class _Index, class _DifferenceType, class _Generator>
+_Index simd_generate_n(_Index __first, _DifferenceType __size, _Generator __g) noexcept {
 __PSTL_USE_NONTEMPORAL_STORES_IF_ALLOWED
 __PSTL_PRAGMA_SIMD
-    for (DifferenceType i = 0; i < size; ++i)
-        first[i] = g();
-    return first + size;
+    for (_DifferenceType __i = 0; __i < __size; ++__i)
+        __first[__i] = __g();
+    return __first + __size;
 }
 
-template<class Index, class BinaryPredicate>
-Index simd_adjacent_find(Index first, Index last, BinaryPredicate pred, bool or_semantic) noexcept {
-    if(last - first < 2)
-        return last;
+template<class _Index, class _BinaryPredicate>
+_Index simd_adjacent_find(_Index __first, _Index __last, _BinaryPredicate __pred, bool __or_semantic) noexcept {
+    if(__last - __first < 2)
+        return __last;
 
-    typedef typename std::iterator_traits<Index>::difference_type difference_type;
-    difference_type i = 0;
+    typedef typename std::iterator_traits<_Index>::difference_type _difference_type;
+    _difference_type __i = 0;
 
 #if __PSTL_EARLYEXIT_PRESENT
     //Some compiler versions fail to compile the following loop when iterators are used. Indices are used instead
-    const difference_type n = last-first-1;
-__PSTL_PRAGMA_VECTOR_UNALIGNED
+    const _difference_type __n = __last - __first-1;
+__PSTL_PRAGMA_VECT__OR_UNALIGNED
 __PSTL_PRAGMA_SIMD_EARLYEXIT
-    for(; i < n; ++i)
-        if(pred(first[i], first[i+1]))
+    for(; __i < __n; ++__i)
+        if(pred(__first[__i], __first[__i + 1]))
             break;
 
-    return i < n ? first + i : last;
+    return __i < __n ? __first + __i : __last;
 #else
     // Experiments show good block sizes like this
     //TODO: to consider tuning block_size for various data types
-    const difference_type block_size = 8;
-    alignas(64) difference_type lane[block_size] = {0};
-    while ( last - first >= block_size ) {
-        difference_type found = 0;
+    const _difference_type __block_size = 8;
+    alignas(64) _difference_type __lane[__block_size] = {0};
+    while ( __last - __first >= __block_size ) {
+        _difference_type __found = 0;
 __PSTL_PRAGMA_VECTOR_UNALIGNED // Do not generate peel loop part
-__PSTL_PRAGMA_SIMD_REDUCTION(|:found)
-        for ( i = 0; i < block_size-1; ++i ) {
+__PSTL_PRAGMA_SIMD_REDUCTION(|:_found)
```

Comment:
There is an error "‘_found’ has not been declared".
I suppose "__found" should be here.